src/ebpf.rs

```diff
@@ -62,6 +62,9 @@ pub const MM_HEAP_START: u64 = 0x300000000;
 /// Start of the input buffers in the memory map
 pub const MM_INPUT_START: u64 = 0x400000000;
 
+/// Alignment of the memory regions in host address space
```

Comment:
Do we really need / want to pass this value around?
Why not hide it inside `src/aligned_memory.rs`?

Reply:
Yeah, I debated that, though maybe we might want different alignments.  For example, we could align our 32k heap on a 32k boundary